source/04_best_practice/custom_model_en.rst

```diff
@@ -0,0 +1,343 @@
+如何自定义神经网络模型（model）
```

Comment:
For English version, it is ok to just be named as `custom_model.rst`, and you should add this file in corresponding `index.rst`

Reply:
Done with [5204a36](https://github.com/opendilab/DI-engine-docs/pull/194/commits/5204a3607a0d3dbba11d1e933651c27165802973) and [3063e43](https://github.com/opendilab/DI-engine-docs/pull/194/commits/3063e430e247650fa13b15febe147128e35c865c)